app/src/main/res/values/strings.xml

```diff
@@ -992,4 +992,23 @@ The new line here must be kept as the second half of the string is clickable for
     <string name="cfr_for_toolbar_delete_icon" moz:removedIn="105" tools:ignore="UnusedResources">Get rid of your personal data, browsing history and more from this session by tapping the trash button. Give it a try!</string>
     <string name="cfr_for_toolbar_delete_icon2">Tap here to trash it all — history, cookies, everything — and start fresh on a new tab.</string>
 
+    <!-- Promote SearchWidget Dialog -->
+
+    !-- Content description (not visible, for screen readers etc.): This is the description for the close button from promote search widget dialog. -->
+    <string name="promote_search_widget_dialog_content_description">Close</string>
+
+    !-- Content description (not visible, for screen readers etc.): This is the description for picture of search widget from promote search widget dialog. -->
+    <string name="promote_search_widget_dialog_picture_content_description">Search widget</string>
+
+    !-- This is the title of promote search widget dialog. -->
+    <string name="promote_search_widget_dialog_title">Browsing history cleared! 🎉</string>
+
+    !-- This is the subtitle of promote search widget dialog. -->
+    <string name="promote_search_widget_dialog_subtitle">We’ll leave you to your private browsing, but get a quicker start next time with the Focus widget on your Home screen.</string>
```

Comment:
Focus being the name of the app and a brand in itself should not be translatable - https://mozilla-l10n.github.io/styleguides/mozilla_general/#brands-copyright-and-trademark
Similar to https://github.com/mozilla-mobile/fenix/pull/27013 `Focus` should be a static string passed as an argument to this longer one (with the string description updated).

Reply:
Done